package.json

```diff
@@ -10,13 +10,13 @@
     "build:chrome": "cross-env VARIANT=chrome webpack --mode=production --config webpack.webext.babel.js",
     "build:firefox": "cross-env VARIANT=firefox webpack --mode=production --config webpack.webext.babel.js",
     "build:safari": "webpack --mode=production --config webpack.safari.babel.js",
-    "watch:chrome": "npm run build:chrome -- --watch",
-    "watch:firefox": "npm run build:firefox -- --watch",
-    "watch:safari": "npm run build:safari -- --watch",
+    "watch:chrome": "run-s build:chrome -- --watch",
+    "watch:firefox": "run-s build:firefox -- --watch",
+    "watch:safari": "run-s build:safari -- --watch",
     "open:chrome": "./script/open-in-chrome ./dist/chrome https://github.com/bitcrowd/tickety-tick",
     "open:firefox": "web-ext run --source-dir ./dist/firefox --pref startup.homepage_welcome_url=https://github.com/bitcrowd/tickety-tick",
-    "bundle:chrome": "cross-env BUNDLE=true npm run build:chrome",
-    "bundle:firefox": "cross-env BUNDLE=true npm run build:firefox",
+    "bundle:chrome": "cross-env BUNDLE=true run-s build:chrome",
+    "bundle:firefox": "cross-env BUNDLE=true run-s build:firefox",
```

Comment:
🤔 isn't the `run-s` from `npm-run-all` usually used to run mutliple npm commands at once or something? 

I am also wondering, since we seem to use `yarn` if this should actually call `yarn` here instead of `npm` anyway (correct me if I am wrong, since I wander on thin ice here :D)

Reply:
Yes, using `yarn` inside the scripts should also work. `run-s` automatically determines whether you're using `npm` or `yarn` to run a script from the `package.json` so I figured I'd just use that and we don't have to bother, even when someone else decides to use `npm run bundle:chrome` or similar 🤷‍♂